packages/components/src/Enumeration/Enumeration.component.js

```diff
@@ -69,29 +66,63 @@ Enumeration.propTypes = {
 	...ItemEditPropTypes,
 };
 
-function ItemsEnumeration({ items, itemsProp, searchCriteria, currentEdit }) {
-	if (items.length > 0) {
-		return (<Items
-			items={items}
-			itemsProp={itemsProp}
-			currentEdit={currentEdit}
-			searchCriteria={searchCriteria}
-		/>);
-	}
-	return null;
+Enumeration.defaultProps = {
+	displayMode: DISPLAY_MODE_DEFAULT,
+	headerLabel: 'Values',
+	emptyLabel: 'This list is empty, click on + to add a value.',
+	toggleAllLabel: 'All',
+	items: [],
+};
+
+function ItemsEnumeration(props) {
+	// Default
+	const { items, itemsProp, emptyLabel } = props;
+	const { toggleAllChecked, toggleAllLabel, onToggleAll } = props;
+	// Edit
+	const { currentEdit } = props;
+	// Search
+	const { searchCriteria } = props;
+	const itemsProps = {
+		items,
+		itemsProp,
+		currentEdit,
+		searchCriteria,
+		toggleAllChecked,
+		toggleAllLabel,
+		onToggleAll,
+		emptyLabel,
+	};
+	return (
+		<Items
+			{...itemsProps}
+		/>
+	);
```

Comment:
I don't think all of that props handling is actually required.
Most of the comments, allocations and object spread can be removed.